google-map.js

```diff
@@ -481,6 +506,38 @@ Polymer({
     this.fire('google-map-ready');
   },
 
+  _initJouneySharingGMap: function() {
+    this.fleetEngineAccessToken = this.$.api.fleetEngineAccessToken;
+    var parent = this;
+    const authTokenFetcher = function() {
+      return {
+        token: parent.fleetEngineAccessToken,
+        expiresInSeconds: 3600
+      };
+    };
```

Comment:
do we need the `parent` variable here? Should work fine if you use an arrow function